frontend/reducers/builds.js

```diff
@@ -0,0 +1,30 @@
+import {
+  buildsFetchStartedType as BUILDS_FETCH_STARTED,
+  buildsReceivedType as BUILDS_RECEIVED,
+  buildRestartedType as BUILD_RESTARTED,
+} from "../actions/actionCreators/buildActions";
+
+const initialState = {
+  isLoading: false,
+}
+
+export default function buildLogs(state = initialState, action) {
```

Comment:
This works because we're now only storing the builds for a particular site in our state and reloading them when visiting a different site.